Description: Fix section of axohelp manual page.
Forwarded: Yes.
Author: Hilmar Preusse <hille42@web.de>
Applied-Upstream: Yes.
Last-Update: 2022-12-31

--- texlive-bin.orig/utils/axodraw2/axodraw2-src/axohelp.1
+++ texlive-bin/utils/axodraw2/axodraw2-src/axohelp.1
@@ -1,4 +1,4 @@
-.TH "axohelp" "1.4" "5 Mar 2021" ""
+.TH "axohelp" 1 "5 Mar 2021" ""
 .SH NAME
 axohelp - helper program for use of LaTeX package axodraw2 with
 pdflatex. 
